spi/bean-decorator/weld/pom.xml

```diff
@@ -77,7 +77,7 @@
                     <archive>
                         <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
                         <manifestEntries>
-                            <Automatic-Module-Name>org.glassfish.soteria.bean.decorator.weld</Automatic-Module-Name>
+                            <Automatic-Module-Name>org.glassfish.soteria.spi.bean.decorator</Automatic-Module-Name>
```

Comment:
For consistency, we should rename the artifactId at the same time to match the new module name.

Reply:
@ggam after updating and testing the SPI, it appeared some more work was needed to get the tests running and the SPI with GF 6. I though we tested it before, but clearly we didn't, as some important exports are missing from the Weld OSGi bundle, so we can't use these. I worked around needing those exports.

Can you take a look again? Thx!